Project setup-MathMagicians

RileyManda opened the pull request:
I have completed the task of Project Setup according to the stipulated [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_setup.md).

The following implementations were made:

- [x] Created application with [react](https://create-react-app.dev/)
- [x] Implemented the setup of  [react /redux linters](https://github.com/microverseinc/linters-config/tree/master/react-redux)

In addition, I have ensured that this project setup fulfills the following requirements:

**General requirements**

- [x] There are [no linter errors](https://github.com/microverseinc/linters-config). 
- [x] Used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Made sure that I have documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**JavaScript requirements**

- [x] Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Project requirements**

- [x] Created the project in a [Node.js](https://nodejs.org/en/) environment
- [x] Created a React application is using [Create React App](https://create-react-app.dev/)
- [x] The application runs in the browser without errors.

Please review my implementation and suggest any improvements and changes.

prolajumokeoni commented on README.md:
```diff
@@ -1 +1,254 @@
+
+## 📝 License <a name="license"></a>
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](./LICENSE)
```
- [ ] Kindly fix the license link in  your readme, it currently shows a  404 error when I click the link 👍🏿

RileyManda replied:
Thank you @prolajumokeoni .
I will make sure to fix this. I see I forgot to include the License in my project and this is resulting in a broken path.

prolajumokeoni commented on README.md:
```diff
@@ -1 +1,254 @@
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
- [ ] Please delete the template placeholder text: `Describe 1 - 3 features you will add to the project.` in your readme, this will make your readme more professional 👍🏿

prolajumokeoni reviewed (changes requested):
Hi @RileyManda 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- React app created with Create React App ✅  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

RileyManda commented:
I have implemented the changes requested by the previous reviewer: @prolajumokeoni 

- [x] Fixed broken MIT licence path in readme.
- [x] Removed instructions text


Please review my implementation and suggest any improvements and changes.

danmainah reviewed (approved):
Hi @RileyManda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Created React Application with Create React App
✔️ Professional written README file with all sections required.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
